src/app/login/login.component.html

```diff
@@ -1,11 +1,13 @@
 <div class="holder">
-  <img src="assets/images/wafb-logo.png" class="wafb-logo">
+  <img src="assets/images/wafb-logo.png" class="wafb-logo" />
   <mat-form-field class="half-width">
     <mat-label>Email</mat-label>
     <input
       matInput
+      type="email"
       placeholder="Ex. john@ex.com"
-      [(ngModel)]="email">
+      [(ngModel)]="email"
+    />
```

Comment:
@drkuster I added a type="email" here so that the browser handles email validation for us.